plugins/core/reporter/api.go

```diff
@@ -108,11 +108,21 @@ var (
 	ConnectionStatusShutdown   ConnectionStatus = 3
 )
 
+type TaskStatus int
+
+const (
+	Pending TaskStatus = iota
```

Comment:
Does this status only work for the trace profiling task? Please add the prefix of the const. In case it conflicts with other concepts.